x/mint/keeper/params.go

```
@@ -1 +1,19 @@
 package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/archway-network/archway/x/mint/types"
+)
+
+// SetParams sets the total set of minting parameters.
+func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
+	k.paramStore.SetParamSet(ctx, &params)
```

I'd add here a panic on invalid parameters.